mobile/src/libs/components/header/header.tsx

```diff
@@ -1,32 +1,28 @@
 import { type NavigationProp } from '@react-navigation/native';
 import React from 'react';
 
-import {
-  Icon,
-  IconButton,
-  Text,
-  TouchableOpacity,
-  View,
-} from '#libs/components/components';
+import { IconButton, Text, View } from '#libs/components/components';
 import { AppColor, ProfileScreenName } from '#libs/enums/enums';
 import { useAppRoute, useNavigation } from '#libs/hooks/hooks';
 import { type ProfileNavigationParameterList } from '#libs/types/types';
 
-import { Badge } from './components/components';
+import { BackButton, Badge } from './components/components';
 import { DEFAULT_BADGE_COUNT } from './libs/constants';
 import { styles } from './styles';
 
 type Properties = {
   title?: string;
   isArrowVisible?: boolean;
   badgeCount?: number;
+  isVisible?: boolean;
   isProfileVisible?: boolean;
 };
 
 const Header: React.FC<Properties> = ({
   title,
   isArrowVisible = false,
   badgeCount = DEFAULT_BADGE_COUNT,
+  isVisible = true,
```

Comment:
what is the difference between isVisible and isArrowVisible? do we need all of these? why it should be with position absolute?

Reply:
isVisible is used to make the header completely transparent (as it should be for the player page), while isArrowVisible is specifically responsible for the back arrow. 

isArrowVisible should have 'position: 'absolute'' because on some pages, the text in the header should be centered. Using 'justifyContent: 'center',' would center both the text and the arrow, but the arrow should always be on the left. 

isVisible has 'position: absolute' because the page uses a gradient, and without 'position: 'absolute'', the header would have its own color, which goes against the design in Figma

Reply:
maybe something like this will work https://medium.com/@athletecoder/react-navigation-gradient-background-fc8d929386df. It's better not to make it position absolute, but to try to set gradient background on navigation level